docs/source/guides/using_dask_entitysets.ipynb

```diff
@@ -0,0 +1,245 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "99052370",
+   "metadata": {},
+   "source": [
+    "# Using Dask EntitySets (BETA)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "84275fe9",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. note::\n",
+    "    Support for Dask EntitySets is still in Beta. While the key functionality has been implemented, development is ongoing to add the remaining functionality.\n",
+    "\n",
+    "    All planned improvements to the Featuretools/Dask integration are `documented on Github <https://github.com/alteryx/featuretools/issues?q=is%3Aopen+is%3Aissue+label%3ADask>`_. If you see an open issue that is important for your application, please let us know by upvoting or commenting on the issue. If you encounter any errors using Dask dataframes, or find missing functionality that does not yet have an open issue, please create a `new issue on Github <https://github.com/alteryx/featuretools/issues>`_."
```

Comment:
> you encounter any errors using Dask dataframes

Might be worth adding a qualifier like "Dask dataframes in EntitySets" or "Dask dataframes with Woodwork typing" because we want them reporting featuretools or woodwork bugs, not dask bugs